.github/workflows/daily_unit_tests_with_nightly_puppet_gem.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
 
       - name: Install the latest nightly build of puppet${{ matrix.puppet_version }} gem
         run: |
-          curl http://nightlies.puppet.com/downloads/gems/puppet${{ matrix.puppet_version }}-nightly/${{ matrix.gem_file }} --output puppet.gem
+          curl htts://nightlies.puppet.com/downloads/gems/puppet${{ matrix.puppet_version }}-nightly/${{ matrix.gem_file }} --output puppet.gem --location
```

Comment:
It looks like you forgot the `p` in `https`

Reply:
Aah oops, I'll fix that now